docs/framework-maria_db_jdbc.md

```diff
@@ -4,18 +4,18 @@ The MariaDB JDBC Framework causes a JDBC driver JAR to be automatically download
 <table>
   <tr>
     <td><strong>Detection Criterion</strong></td>
-    <td>Existence of a single bound MariaDB or MySQL service and no provided MariaDB or MySQL JDBC JAR.
+    <td>Existence of a single bound MariaDB or MySQL service and NO provided MariaDB or MySQL JDBC jar.
       <ul>
-        <li>Existence of a MariaDB service is defined as the <a href="http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES"><code>VCAP_SERVICES</code></a> payload containing a service who's name, label or tag has <code>mariadb</code> as a substring.</li>
-        <li>Existence of a MySQL service is defined as the <a href="http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES"><code>VCAP_SERVICES</code></a> payload containing a service who's name, label or tag has <code>mysql</code> as a substring.</li>
-        <li>Existence of a MariaDB JDBC JAR is defined as the application containing a JAR who's name matches <tt>mariadb-java-client*.jar</tt></li>
-        <li>Existence of a MySQL JDBC JAR is defined as the application containing a JAR who's name matches <tt>mysql-connector-java*.jar</tt></li>
+        <li>Existence of a MariaDB service is defined as the <a href="http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES"><code>VCAP_SERVICES</code></a> payload containing a service whose name, label or tag has <code>mariadb</code> as a substring.</li>
+        <li>Existence of a MySQL service is defined as the <a href="http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES"><code>VCAP_SERVICES</code></a> payload containing a service whose name, label or tag has <code>mysql</code> as a substring.</li>
+        <li>NON Existence of a MariaDB JDBC jar is defined as the application containing a JAR whose name matches <code>mariadb-java-client*.jar</code></li>
```

Comment:
Probably not the clearest but these 2 bullets are not detection-criteria, rather definitions of the single detection-criteria sentence at the top of the table cell.

So they are correct without the 'NON', because they are trying to say: 'existence of X is defined as Y'.